doc/source/whatsnew/v1.0.0.rst

```diff
@@ -317,6 +317,7 @@ I/O
 - Bug in :func:`read_hdf` closing stores that it didn't open when Exceptions are raised (:issue:`28699`)
 - Bug in :meth:`DataFrame.read_json` where using ``orient="index"`` would not maintain the order (:issue:`28557`)
 - Bug in :meth:`DataFrame.to_html` where the length of the ``formatters`` argument was not verified (:issue:`28469`)
+- Added ``encoding`` argument to :func:`DataFrame.to_string` for non-ascii text (:issue:`28766`)
```

Comment:
can you move to Other enhancements section

Reply:
Sure @simonjayhawkins . I'll make the necessary changes.

Reply:
can you move to Other enhancements section (starts on L99)